dvc.nuspec

```diff
@@ -45,7 +45,7 @@ as well as ad-hoc data file suffixes and prefixes.
 ]]></description>
     <releaseNotes>https://github.com/iterative/dvc/releases/latest</releaseNotes>
     <dependencies>
-      <dependency id="Python" version="[3.8, 3.9]" />
+      <dependency id="Python" version="[3.8, 3.12)" />
```

Comment:
> ```
> <!-- Accepts 1.3.2 up to 1.4.x, but not 1.5 and higher. -->
> <dependency id="ExamplePackage" version="[1.3.2,1.5)" />
> ```

I found this in See https://learn.microsoft.com/en-us/nuget/concepts/package-versioning.